control-plane-operator/controllers/hostedcontrolplane/oauth/params.go

```diff
@@ -48,6 +48,7 @@ type OAuthConfigParams struct {
 	ExternalHost             string
 	ExternalPort             int32
 	ServingCert              *corev1.Secret
+	NamedCertificates        []configv1.APIServerNamedServingCert
```

Comment:
How do they get into the cluster namespace?

Reply:
Reconciled by the hosted cluster references secret reference

Reply:
Then those certs are passed as a secret reference on the hosted cluster object and it will reconcile